packages/venia-concept/src/components/CmsBlock/cmsBlock.js

```diff
@@ -0,0 +1,57 @@
+import { Component, createElement } from 'react';
+import { PropTypes } from 'prop-types';
+import { Query } from 'react-apollo';
+import gql from 'graphql-tag';
+import classify from 'src/classify';
+import defaultClasses from './cmsBlock.css';
+
+const cmsBlockQuery = gql`
+    query cmsBlocks($identifiers: [String]!) {
+        cmsBlocks(identifiers: $identifiers) {
+            items {
+                content
+            }
+        }
+    }
+`;
+
+class CmsBlock extends Component {
+    static propTypes = {
+        classes: PropTypes.shape({
+            root: PropTypes.string,
+            content: PropTypes.string
+        }),
+        identifiers: PropTypes.oneOfType([PropTypes.string, PropTypes.array])
+    };
+
+    render() {
+        const { identifiers, classes } = this.props;
+        return (
+            <div className={classes.root}>
+                <Query query={cmsBlockQuery} variables={{ identifiers }}>
+                    {({ loading, error, data }) => {
+                        if (error) return <div>Data Fetch Error</div>;
+                        if (loading) return <div>Fetching Data</div>;
+                        if (data.cmsBlocks.items == '')
+                            return <div>Here is not any block(s)</div>;
```

Comment:
It might be better to log something to the console and return null, don't you think? Should a user see this error?